src/lib/errors.ts

```diff
@@ -23,6 +23,12 @@ interface CodedSysError extends Error {
 	code?: string;
 }
 
+export interface ChildProcessError extends Error {
```

Comment:
We should try to move towards making modules more self contained. This error could probably be declared inside the `update-lock` module, where it is going to be used.

Even better it could be declared in the `lockfile` module and the lock function could catch an error on the exec and return a `LockfileExistsError` if `code == 73` or a generic error otherwise.

Reply:
Done, moved to inside lockfile. The Supervisor-specific "Lockfile exists for { service: ..., appId: ... }" message creation is still in update-lock.ts though.